packages/cli/src/commands/upgrade/__tests__/upgrade.test.js

```diff
@@ -172,7 +172,24 @@ success Upgraded React Native to v0.58.4 🎉. Now you can review and commit the
 
 test('fetches regular patch, adds remote, applies patch, installs deps, removes remote,', async () => {
   (fetch: any).mockImplementation(() => Promise.resolve(samplePatch));
-  await upgrade.func([newVersion], ctx, opts);
+  await upgrade.func(
+    [newVersion],
+    {
+      ...ctx,
+      project: {
+        ...ctx.project,
```

Comment:
You could use `merge` util for deep merge. Doing it this way can execute lazy getters, which is not necessarily bad, but why not.